### PR TITLE
DM-31000: Hide version.py from mypy

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -9,12 +9,16 @@ exclude =
   **/*/version.py,
   tests/.tests
 
-[tool:pytest]
+[tool:pytest] --ignore-glob=*version.py
 addopts = --flake8 --black --mypy
```
Comment (wvreeven) [inline setup.cfg:13]: Shouldn't the ignore bit in the previous line be on this line instead?

**Review by wvreeven — CHANGES_REQUESTED** [review]
I probably spotted an issue with setup.cfg so please correct that.

File: setup.cfg (hunk 1 above)
Comment (r-owen) [reply setup.cfg:13]: yes. Thanks for catching that.

**Review by wvreeven — APPROVED** [review]
Looks great now, thanks.

**Pull request merged**